test/lib/validation.go

```diff
@@ -157,3 +159,44 @@ func (client *Client) getContainerName(podName, namespace string) (string, error
 	containerName := pod.Spec.Containers[0].Name
 	return containerName, nil
 }
+
+// CheckConfigMapsExist will check if copy configmaps exist.
+func (client *Client) CheckConfigMapsExist(namespace string, names ...string) error {
+	return wait.PollImmediate(interval, timeout, func() (bool, error) {
+		for _, name := range names {
+			_, err := client.Kube.Kube.CoreV1().ConfigMaps(namespace).Get(name, metav1.GetOptions{})
+			if k8serrors.IsNotFound(err) {
+				return false, nil
+			} else if err != nil {
+				return false, err
+			}
+		}
+		return true, nil
+	})
+}
+
+// CheckConfigMapsEqual will check if configmaps have the same data as the original one.
+func (client *Client) CheckConfigMapsEqual(originalNamespace, cmp string, names ...string) error {
+	return wait.PollImmediate(interval, timeout, func() (bool, error) {
+		for _, name := range names {
+			// Get original configmap
+			origianlCM, err := client.Kube.Kube.CoreV1().ConfigMaps(originalNamespace).Get(name, metav1.GetOptions{})
+			if k8serrors.IsNotFound(err) {
```

Comment:
As calling `CheckConfigMapsExist` before `CheckConfigMapsEqual` will make sure the configmap exists, it seems we can return `false, err` whenever there is an error?